tests/MessageTest.php

```diff
@@ -605,6 +605,9 @@ public function provideDateCases(): array
             ['2007-04-06T12:37:39+0000', 'Fri Apr 06 12:37:39 2007'],
             ['2008-02-12T06:35:05+0000', '12 Feb 2008 06:35:05 UT +0000'],
             ['2010-07-09T21:40:33+0000', 'Fri, 9 Jul 2010 21:40:33 UT'],
+            ['2014-09-15T05:25:04+0000', 'Mon, 15 09 2014 05:25:04'],
```

Comment:
My Thunderbird 68 doesn't handle this kind of format: as long as all other tests pass it's ok for me to handle this, but please add a comment and the end of the line that isn't regular, so we can safely remove support for this type if it will ever be needed
```suggestion
            ['2014-09-15T05:25:04+0000', 'Mon, 15 09 2014 05:25:04'], # Non compliant to RFC2822#section-3.3
```